testing/debconf/APKBUILD

```diff
@@ -0,0 +1,55 @@
+# Contributor: TBK <alpine@jjtc.eu>
+# Maintainer: TBK <alpine@jjtc.eu>
+pkgname=debconf
+pkgver=1.5.71
+pkgrel=0
+pkgdesc="Debconf is a configuration management system for Debian packages."
+url="https://packages.debian.org/debconf"
+arch="noarch"
+license="BSD-2-Clause"
+options="!check" # Test suite is broken. CopyDBTestSetup is missing new method
+depends="perl"
+makedepends="findutils po4a"
+subpackages="$pkgname-doc $pkgname-bash-completion:bashcomp:noarch"
+source="https://deb.debian.org/debian/pool/main/d/debconf/debconf_$pkgver.tar.xz
+	fix-makefile_destdir.patch
+	"
+builddir="$srcdir/$pkgname"
+
+build() {
+	cd "$builddir"
```

Comment:
These are redundant nowadays, since abuild automatically cd's to `$builddir`

Reply:
@Ikke fixed. I have expanded my Makefile patch so there are two new subpkgs to mirror the Debian offers. The only difference is that Debians' debconf-utils contains the utils related man pages, I package them in debconf-doc.

I will upstream the patch.